File: app/src/main/java/de/bitb/astroskop/ui/main/circumstances/ICircumstancesView.java
package de.bitb.astroskop.ui.main.circumstances;

import de.bitb.astroskop.model.Circumstance;
import de.bitb.astroskop.ui.base.IBaseView;

public interface ICircumstancesView extends IBaseView {

    void refreshView();

    void openCircumstanceDetails(Circumstance circumstance);

    void openDeleteDialog(Circumstance circumstance);
}

File: app/src/main/java/de/bitb/astroskop/ui/main/circumstances/ICircumstancesView.kt
package de.bitb.astroskop.ui.main.circumstances

import de.bitb.astroskop.model.Circumstance
import de.bitb.astroskop.ui.base.IBaseView

interface ICircumstancesView : IBaseView {

    fun refreshView()

    fun openCircumstanceDetails(circumstance: Circumstance)

    fun openDeleteDialog(circumstance: Circumstance)
}
